pkgs/by-name/an/anytype/tantivy-go/default.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tantivy-go";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "anyproto";
+    repo = pname;
```

Do not specify repo name by the package name

It is considered an antipattern, AFAIK.
```suggestion
    repo = "tantivy-go";
```